packages/iframe-coordinator-cli/package.json

```diff
@@ -1,46 +1,34 @@
 {
-  "name": "ifc-cli",
-  "version": "4.6.1",
+  "name": "iframe-coordinator-cli",
+  "version": "5.0.0-beta.0",
   "description": "CLI for local iframe-coordinator development",
   "dependencies": {
-    "@webcomponents/custom-elements": "^1.2.4",
     "cheerio": "^1.0.0-rc.10",
     "commander": "^2.20.3",
-    "core-js": "^2.6.12",
     "custom-event-polyfill": "^1.0.7",
     "dev-cert-authority": "^1.1.1",
     "express": "^4.17.3",
     "find-root": "^1.1.0",
     "http-proxy-middleware": "^1.3.1",
     "iframe-coordinator": "file:../iframe-coordinator",
-    "vue": "^2.6.10",
+    "vue": "^2.7.10",
     "vue-class-component": "^7.0.2",
     "vue-notification": "^1.3.16",
     "vue-property-decorator": "^8.1.0",
     "vue-router": "^3.0.3",
     "vuex": "^3.0.1"
   },
   "devDependencies": {
-    "@babel/preset-env": "^7.4.5",
-    "@types/chai": "^4.1.0",
```

Comment:
I removed `types/chai`, `types/mocha` and `chai`. There were errors when testing caused by having duplicate identifiers with `jasmine`. (similar issue is described here https://github.com/angular/angular-cli/issues/3518) The types seemed to only be used in an example test, so I removed the example test and types.